handlers/node_api.py

```diff
@@ -0,0 +1,77 @@
+import json
+from typing import Optional
+
+from aws_lambda_context import LambdaContext
+
+from usecases.node_api import NodeApi
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def get_address_balance(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    node_api: Optional[NodeApi] = None
+) -> dict:
+    node_api = node_api or NodeApi()
+    address = event.query.get("address")
+    if address is None:
+        raise Exception("invalid_parameters")
+
+    response = node_api.get_address_balance(address)
+    if response is None:
+        return {
+            "statusCode": 503,
```

Comment:
Why 503?

Reply:
The only error (besides timeout) that I found this api returning is 403 ("This path is forbidden in this server"). So a "Service Unavailable" seems more intuitive to return.
We could also do a refactor to make the `HathorCoreClient` always return the `requests` response, so we can bubble up the full-node response and return it to the client.